scipy/signal/splinemodule.c

```diff
@@ -45,13 +45,33 @@ convert_strides(npy_intp* instrides,npy_intp* convstrides,int size,int N)
 
 static char doc_cspline2d[] = "cspline2d(input {, lambda, precision}) -> ck\n"
 "\n"
```

Comment:
The signature still doesn't show up in the doc for all three functions you're touching here. You could fix that by getting the format right here. See for example `convolve2d`, which also has its docs in C and the signature does show up.

Reply:
Thanks. I agree. I fixed it.